h_program-lang/Error_code.ml

```diff
@@ -405,6 +405,15 @@ let try_with_print_exn_and_reraise file f =
     (* does not really re-raise :( lose some backtrace *)
     raise exn
 
+(* fast = no stack trace *)
+let try_with_print_exn_and_exit_fast file f =
+  try
+    f ()
+  with exn ->
```

Comment:
If you don't need the stack trace for some exception you can also use `raise_notrace`.

Reply:
What will `get_backtrace ()` do then when it gets called? The manual says nothing, about it, I'm filing a ticket.

Reply:
I guess the empty string, but it would be great if it was specified!

Reply:
https://github.com/ocaml/ocaml/issues/10467